addon/components/lmb-plugin/list.hbs

```diff
@@ -37,6 +37,9 @@
     </tr>
   </:header>
   <:body>
+    {{#if @error}}
+      <Common::Search::AlertLoadError @error={{@error}} />
+    {{/if}}
```

Comment:
I think it might be better to not show the table at all when there is an error? We could simply show the error without the table headers being shown.

Reply:
I did this as I was just mimicking the existing API, which passed the error to the list component, even though it was never used. We could also just not show the List at all on error.